OpenXmlPowerTools/WmlToHtmlConverter.cs

```diff
@@ -2370,66 +2369,33 @@ private static int CalcWidthOfRunInTwips(XElement r)
                 runText = sb.ToString();
             }
 
+            int w;
+            const decimal dpi = 96m;
+            const FontStyle fs2 = FontStyle.Regular;
+            const FontStyle fs3 = FontStyle.Bold;
+            var ff2 = new FontFamily("Times New Roman");
+
             try
             {
-                using (Font f = new Font(ff, (float) sz/2f, fs))
-                {
-                    const TextFormatFlags tff = TextFormatFlags.NoPadding;
-                    var proposedSize = new Size(int.MaxValue, int.MaxValue);
-                    var sf = TextRenderer.MeasureText(runText, f, proposedSize, tff);
-                        // sf returns size in pixels
-                    const decimal dpi = 96m;
-                    var twip = (int) (((sf.Width/dpi)*1440m)/multiplier + tabLength*1440m);
-                    return twip;
-                }
+                w = MetricsGetter.GetTextWidth(ff, fs, sz, runText);
```

Comment:
As above, all errors (and fallbacks), should be handled in the `GetTextWidth` method